src/renderer/screens/accounts/index.js

```diff
@@ -83,17 +82,8 @@ const AccountsPage = ({
     [history],
   );
 
-  if (accounts.length === 0) {
-    return (
-      <>
-        <TrackPage category="Accounts" accountsLength={accounts.length} />
-        <TopBannerContainer>
-          <UpdateBanner />
-          <MigrationBanner />
-        </TopBannerContainer>
-        <EmptyState />
-      </>
-    );
+  if (!accounts.length) {
+    return <Redirect to="/" />;
```

Comment:
I don't know about this. I think it's not good to be redirected to Portfolio when for instance, you just did a "Remove" on the last account.

Reply:
same